examples/ecommerce/src/App.vue

```diff
@@ -78,14 +71,29 @@
                 </div>
               </template>
             </ais-clear-refinements>
+
+            <ais-stats data-layout="mobile">
```

Comment:
The text size is smaller than on the design on this widget.

Reply:
5d72727